static/app/views/onboarding/useConfigureSdk.tsx

```diff
@@ -127,7 +132,7 @@ export function useConfigureSdk({
         }
       );
     },
-    [createPlatformProject, onboardingContext, organization]
+    [createPlatformProject, onboardingContext, organization, createProject]
```

Comment:
Can we add a test for this to prevent this from happening again? There is a similar [test](https://github.com/getsentry/sentry/blob/6f7a880073bafad053082643bf6fd506f2ca9a05/static/app/components/onboarding/frameworkSuggestionModal.spec.tsx#L76)